docker/Dockerfile

```diff
@@ -111,7 +111,10 @@ ENV PIP_NO_CACHE_DIR=off \
     SENTRY_FILESTORE_DIR=/var/lib/sentry/files \
     # Disable some unused uWSGI features, saving dependencies
     # Thank to https://stackoverflow.com/a/25260588/90297
-    UWSGI_PROFILE_OVERRIDE=ssl=false;xml=false;routing=false
+    UWSGI_PROFILE_OVERRIDE=ssl=false;xml=false;routing=false \
+    # UWSGI STATS
+    UWSGI_ENABLE_METRICS=true \
```

Comment:
I think it's fine that we don't set this here. We can explicitly set it if we want it on. Just as long as the plugin is installed.

No real reason to have overhead of metrics if they aren't being emitted anywhere.